mnefun/_mnefun.py

```diff
@@ -1223,7 +1223,7 @@ def _load_trans_to(p, subj, run_indices, raw=None):
                                subj + '_twa_pos.fif')
             if not op.isfile(trans_to):
                 calc_twa_hp(p, subj, trans_to, run_indices)
-        trans_to = mne.read_trans(trans_to)
+        trans_to = mne.read_trans(p.trans_to)
```

Comment:
this isn't the right fix, it should be:
```
else:
    trans_to = p.trans_to
```
Otherwise the conditionals that precede this line are useless